client/src/app/page.module.css

```diff
@@ -4,8 +4,62 @@
     justify-content: space-between;
     align-items: center;
     padding: 6rem;
-    min-height: 100vh;
+    min-height: 100%;
     /* background-color: #A9CBBC; */
   }
-  
+
+.landing {
```

Comment:
I think it would be a good idea to make this a flexbox container so you can position all the elements inside without using width/margins for the message, image, etc.